templates/spring-kotlin/files/.platform.app.yaml

```diff
@@ -0,0 +1,30 @@
+# This file describes an application. You can have multiple applications
+# in the same project.
+#
+# See https://docs.platform.sh/user_guide/reference/platform-app-yaml.html
+
+# The name of this app. Must be unique within a project.
+name: app
+
+# The runtime the application uses.
+type: "java:8"
+
+disk: 1024
+
+# The hooks executed at various points in the lifecycle of the application.
+hooks:
+    build: mvn clean install
+
+
+# The relationships of the application with services or other applications.
+#
+# The left-hand side is the name of the relationship as it will be exposed
+# to the application in the PLATFORM_RELATIONSHIPS variable. The right-hand
+# side is in the form `<service name>:<endpoint name>`.
+#relationships:
+#    redis: 'redis:redis'
```

Comment:
An odd service to have as the sample default.  Esp. with the old-style naming.  Shall we use `database` as the example here instead?